shared/lib-bie-kafka/src/main/java/gov/va/vro/model/biekafka/test/BieMessagePayloadFactory.java

```diff
@@ -45,12 +45,13 @@ private static void setDeletedPayloadValues(BieMessagePayload payload) {
 
   public static BieMessagePayload create() {
     // generate a random contention classification event type
-    ContentionEvent eventType = faker.options().option(ContentionEvent.values());
+    //    ContentionEvent eventType = faker.options().option(ContentionEvent.values());
+    ContentionEvent eventType = ContentionEvent.CONTENTION_ASSOCIATED;
```

Comment:
this appears to be a slight adjustment to the test behavior, changing the event type from random selection to `CONTENTION_ASSOCIATED` - does that align with your intent? if so, would recommend deleting the comment and the commented-out `faker` code (L47-48).

Reply:
Oops, yes, I will revert this as well. It was intended to force the `eventType` to be of this type only, as opposed to being set in a randomized fashion.